internal/querynodev2/tasks/query_task.go

```diff
@@ -125,6 +126,10 @@ func (t *QueryTask) Execute() error {
 		return err
 	}
 
+	relatedDataSize := lo.Reduce(querySegments, func(acc int64, seg segments.Segment, _ int) int64 {
+		return acc + seg.MemSize()
```

Comment:
note that the actual data size will be larger than on disk size if we use the memory size